app/javascript/components/patient/Patient.js

```diff
@@ -163,85 +223,176 @@ class Patient extends React.Component {
                 </Row>
                 <Row>
                   <Col className="text-truncate">
-                    <span>
-                      {this.props.details.address_line_1 && `${this.props.details.address_line_1}`}
-                      {this.props.details.address_line_2 && ` ${this.props.details.address_line_2}`}
-                      {this.props.details.foreign_address_line_1 && `${this.props.details.foreign_address_line_1}`}
-                      {this.props.details.foreign_address_line_2 && ` ${this.props.details.foreign_address_line_2}`}
-                    </span>
-                    <br />
-                    <span>
-                      {this.props.details.address_city ? this.props.details.address_city : ''}
-                      {this.props.details.address_state ? ` ${this.props.details.address_state}` : ''}
-                      {this.props.details.address_county ? ` ${this.props.details.address_county}` : ''}
-                      {this.props.details.address_zip ? ` ${this.props.details.address_zip}` : ''}
-                      {this.props.details.foreign_address_city ? this.props.details.foreign_address_city : ''}
-                      {this.props.details.foreign_address_country ? ` ${this.props.details.foreign_address_country}` : ''}
-                      {this.props.details.foreign_address_zip ? ` ${this.props.details.foreign_address_zip}` : ''}
-                    </span>
-                    <br />
+                    <Row className="py-1">
+                      <Col>
+                        <b>HOME ADDRESS</b>
+                      </Col>
+                    </Row>
+                    {(this.props.details.address_line_1 ||
+                      this.props.details.address_line_2 ||
+                      this.props.details.address_city ||
+                      this.props.details.address_state ||
+                      this.props.details.address_zip) && (
+                      <React.Fragment>
+                        <Row>
+                          <Col>
+                            <b>Address 1:</b> <span>{this.props.details.address_line_1 ? `${this.props.details.address_line_1}` : '--'}</span>
```

Comment:
Why the string interpolation here? Couldnt this just be 
`{this.props.details.address_line_1 || '--'}`

Reply:
fixed all instances of this - they were copied over